kratos/sources/kernel.cpp

```diff
@@ -21,15 +21,15 @@
 #include "input_output/logger.h"
 
 namespace Kratos {
-Kernel::Kernel() : mpKratosCoreApplication(Kratos::make_shared<KratosApplication>(
-                std::string("KratosMultiphysics"))) {
+Kernel::Kernel(bool IsDistributedRun) : mpKratosCoreApplication(Kratos::make_shared<KratosApplication>(
+                std::string("KratosMultiphysics"))), mIsDistributedRun(IsDistributedRun) {
```

Comment:
a minor comment, however i would propose to have after the logo an output tellign that we are running indistributed mode (otherwise it is hard to tell by just seeing the run)